src/content_mapblock.cpp

```diff
@@ -172,6 +172,13 @@ static inline void getNeighborConnectingFace(v3s16 p, INodeDefManager *nodedef,
 		*neighbors |= v;
 }
 
+namespace
```

Comment:
What's wrong with being idiomatic? Consistency is important, I suppose. (Although static does only work with functions, not classes and types etc.) Will make change, but don't understand the rationale.